### PR TITLE
[Feat.] Added setOwnerReference in Job Builder

### PR DESCRIPTION
**Pull request by rahulchheda**
Signed-off-by: Rahul M Chheda <rahul.chheda@mayadata.io>

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]

File: kubernetes/job/build.go [hunk 1]
```diff
@@ -19,9 +19,9 @@ package job
 import (
 	"errors"
 	"fmt"
-
```
Comment (chandankumar4) [inline kubernetes/job/build.go:22]: This empty line is required

Comment (chandankumar4) [reply kubernetes/job/build.go:22]: It should be something like:
```
	"errors"
	"fmt"

	batchv1 "k8s.io/api/batch/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"

	jobspec "github.com/litmuschaos/kube-helper/kubernetes/jobspec"
```

**Review by chandankumar4 — APPROVED** [review]
LGTM

**Review by ksatchit — APPROVED** [review]

**Comment by ksatchit** [comment]
@rahulchheda let us know if we can merge this/ needs any other changes to be pulled

**Pull request merged**